src/hist/basehist.py

```diff
@@ -630,3 +630,43 @@ def sum(self, flow: bool = False) -> float | bh.accumulators.Accumulator:
             raise AssertionError(f"Unsupported storage type {storage}")
 
         return super().sum(flow=flow)
+
+    def chisquare_1samp(
+        self,
+        distribution: str | Callable[..., Any],
+        flow: bool = False,
+        args: Any = (),
+        kwds: Any = None,
```

Comment:
```suggestion
        *,
        flow: bool = False,
        *args: Any,
        **kwargs: Any,
```

Reply:
Though are these needed? What args might one pass through?

Reply:
I think you could do this with `functools.partial` on the function you pass in to the distribution, right? Would be nice to keep our API surface low.

Reply:
Yes these are supposed to be used as function arguments for the CDF which may be a cdf from a scipy distribution, a numba-stats distribution or a user-defined CDF. It did it like that because that's similar to the scipy API (`scipy.stats.ks_1samp`). I didn't put much thought into it.